st2common/st2common/content/utils.py

```diff
@@ -295,7 +295,11 @@ def get_pack_file_abs_path(
     path_components.append(normalized_file_path)
     result = os.path.join(*path_components)  # pylint: disable=E1120
 
-    assert normalized_file_path in result
+    if normalized_file_path not in result:
+        raise ValueError(
+            f"This is not a normalized path {normalized_file_path}"
+            f"to prevent directory traversal {result}."
```

Comment:
Subtle typo - just needs an extra space.

```suggestion
            f" to prevent directory traversal {result}."
```